tests/wpt/mozilla/tests/mozilla/bluetooth/getCharacteristic/get-same-characteristic.html

```diff
@@ -4,22 +4,25 @@
 <script src="/bluetooth/bluetooth-helpers.js"></script>
 <script>
 'use strict';
-promise_test(t => {
+promise_test(() => {
     window.testRunner.setBluetoothMockDataSet(adapter_type.heart_rate);
-    return window.navigator.bluetooth.requestDevice({filters: [{services: [heart_rate.name]}], optionalServices: [generic_access.name]})
+    return window.navigator.bluetooth.requestDevice({
+        filters: [{services: [heart_rate.name]}],
+        optionalServices: [generic_access.name]
+    })
     .then(device => device.gatt.connect())
-    .then(gattServer =>
-      gattServer.getPrimaryService(generic_access.name))
+    .then(gattServer => gattServer.getPrimaryService(generic_access.name))
     .then(service => {
-    	return Promise.all([
-          service.getCharacteristic(device_name.name),
-          service.getCharacteristic(device_name.name)])
-    	.then(characteristics => {
-          for (let i = 1; i < characteristics.length; i++) {
-            assert_not_equals(characteristics[0], characteristics[i],
-                          'Should return the same characteristic as the first call.');
-          }
-      });
-    });
+        return Promise.all([
+            service.getCharacteristic(device_name.name),
+            service.getCharacteristic(device_name.name)])
+        .then(characteristics => {
+            for (let i = 1; i < characteristics.length; i++) {
+                assert_not_equals(characteristics[0], characteristics[i],
```

Comment:
Why is there `assert_not_equles`, when the description says: 'Should return the same characteristic as the first call.'